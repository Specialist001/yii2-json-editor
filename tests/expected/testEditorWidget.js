dataJsonEditor_729ee6af = new JSONEditor(document.getElementById('data-json-editor'), {"modes":["code","form","text","tree","view"],"mode":"view","onChange":function() {jQuery('#data').val(dataJsonEditor_729ee6af.getText()); var userFunction = function () {console.log(this);}; userFunction.call(this);},"onError":function (error) {console.log(error);},"onModeChange":function(newMode, oldMode) {if (["view"].indexOf(newMode) !== -1) {dataJsonEditor_729ee6af.collapseAll();}if (["tree","form"].indexOf(newMode) !== -1) {dataJsonEditor_729ee6af.expandAll();} var userFunction = function (nMode, oMode) {console.log(this, nMode, oMode);}; userFunction.call(this, newMode, oldMode);}}, {});
jQuery('#data').parents('form').submit(function() {jQuery('#data').val(dataJsonEditor_729ee6af.getText());});
dataJsonEditor_729ee6af.collapseAll();