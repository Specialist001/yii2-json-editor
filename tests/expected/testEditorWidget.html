<input type="hidden" id="data" name="data" value="{}" data-json-editor-name="dataJsonEditor_729ee6af"><div id="data-json-editor" class="container"></div>